clipper/src/main.py

```diff
@@ -209,6 +209,8 @@ def option_parser():
     # parser.add_option("--reverse_strand", dest="reverse_strand",default=False, action="store_true", help="adds option to reverse strand")
     parser.add_option("--timeout", dest="timeout", default=None, type=int,
                       help="adds timeout (in seconds) to genes that take too long (useful for debugging only, or if you don't care about higly expressed genes)")
+    parser.add_option("--datadir", dest="datadir", default=None, type=str,
+                      help="folder that stores *AS.STRUCTURE.gff") #TODO find a default place to store
```

Comment:
Is AS.STRUCTURE.gff the only file needed? I thought we also needed _genes.bed and _exons.bed? Or maybe those were deprecated/not used anymore...

Reply:
they are still used. we need the regions/* too
so all the files needed are:
1. `DATA_DIR/{prefix}*AS.STRUCTURE.gff`
2. `DATA_DIR/regions/{prefix}*

let me clear that up